### DISCUSSION TITLE
_nempty fails on json field type

### DISCUSSION THREAD

**Discussion by tonxxd** [directus/directus#6162] [Archive]
1) The issue and what you expected to happen
    Using filter `json_column: {_nempty: true}` fails with error `operator does not exist: json <> unknown`

2) Exact steps to reproduce this issue
    Create json field and try to filter non empty values

**Comment by rijkvanzanten** [comment]
JSON in Postgres can never be an empty string. Use `nnull` instead 👍🏻
